File: Week-02/Day-1/Greet.ts
// -  Create variable named `name` and assign the value `Greenfox` to it
// -  Create a function called `greet` that greets it's input parameter
//     -  Greeting is printing e.g. `Greetings, dear Greenfox`
//     -  Prepare for the special case when no parameters are given
// -  Greet `name`

let myName: string = 'Greenfox';

function greet(x?: string) {
  if (x === undefined) {
    x = 'name'
    console.log('Greetings, dear ' + x);
  } else {
    console.log('Greetings, dear ' + x);
  }

}

greet(myName);

File: Java/Week2/Day1/Greet.java
//  From now on, create the usual class wrapper
//  and main method on your own.
// - Create a string variable named `al` and assign the value `Green Fox` to it
// - Create a function called `greet` that greets it's input parameter
//     - Greeting is printing e.g. `Greetings dear, Green Fox`
// - Greet `al`

/**
 * Greet
 */
public class Greet {
  public static void main(String[] args) {
    String al = "Green fox";
    greetFunction(al);
  }

  public static void greetFunction(String name) {
    System.out.println("Greetings dear, " + name);
  }
}